var/spack/repos/builtin/packages/eccodes/package.py

```diff
@@ -50,6 +50,7 @@ class Eccodes(CMakePackage):
     license("Apache-2.0")
 
     version("develop", branch="develop")
+    version("2.34.0", sha256="81c00aeb3c65ee0c3060df3932ab0e00f92bd20edcd3bd6960ba6ce3159bce14")
```

Comment:
```
$ spack checksum eccodes 2.34.0 
==> Found 1 version of eccodes
==> Fetching https://confluence.ecmwf.int/download/attachments/45757960/eccodes-2.34.0-Source.tar.gz?api=v2

    version("2.34.0", sha256="3cd208c8ddad132789662cf8f67a9405514bfefcacac403c0d8c84507f303aba")
```

Reply:
Thanks for spotting. Fixed in https://github.com/spack/spack/pull/42618/commits/043436d9fce397189e57d96c9ab770bc32d595b3